lib/src/components/mixins/hitbox.dart

```diff
@@ -11,10 +11,6 @@ mixin Hitbox on PositionComponent {
   UnmodifiableListView<HitboxShape> get shapes => UnmodifiableListView(_shapes);
 
   void addShape(HitboxShape shape) {
-    assert(
```

Comment:
I am a little confused on why this existed on the first place and why we are removing it can you shed some light on this?

cc @spydon

Reply:
I wasn't sure about what should be late, ?, etc on the shapes stuff

Reply:
I'm on my phone, and can only see the assert line, not what is actually in it. But I'm guessing it is checking if the shape.component is null?
It is because a shape can only be attached to one component, so if it is trying to be attached again to another component it shouldn't be allowed.